DependencyInjection/Configuration.php

```diff
@@ -61,6 +61,14 @@ public function getConfigTreeBuilder()
                                 ->end()
                             ->end()
                         ->end()
+                        ->arrayNode('to')
+                            ->prototype('array')
+                                ->children()
+                                    ->scalarNode('address')->cannotBeEmpty()->isRequired()->end()
+                                    ->scalarNode('name')->end()
+                                ->end()
+                            ->end()
+                        ->end()
                         ->scalarNode('recipient_address')->isRequired()->cannotBeEmpty()->end()
```

Comment:
I prefer a BC break: remove the `recipient_address` node and update the `CHANGELOG.md` with this new feature. I will create a new tag before merging this PR.
So the `to` node must be required.